lib/src/platform_client.dart

```diff
@@ -433,6 +433,26 @@ class PlatformClient {
         body: jsonEncode({'userId': _userId, 'photoIds': ids}),
       );
 
+  /// Registers the device's push token so that it can receive push notifications.
+  ///
+  /// `token` is the Base64-encoded Apple Push Notification service (APNs) device token on iOS or the Firebase Cloud
+  /// Messaging (FCM) registration token on Android. It can be obtained using the
+  /// [`plain_notification_token`](https://pub.dev/packages/plain_notification_token) package.
+  Future<void> registerPushToken(String token) async {
+    _verifyIsLoggedIn();
+
+    if (kIsWeb || !(Platform.isAndroid || Platform.isIOS)) {
+      throw UnsupportedError('Unsupported platform');
+    }
+
+    String body = jsonEncode({
+      'token': token,
+      'type': Platform.isAndroid ? 'ANDROID' : 'IOS',
```

Comment:
would be nice to check if the platform is supported... I don't think that expecting that all users of this API will check this appropriately is super wise.

Reply:
I don't understand this comment. Above, we throw an `UnsupportedError` if the platform is not supported:
```dart
    if (kIsWeb || !(Platform.isAndroid || Platform.isIOS)) {
      throw UnsupportedError('Unsupported platform');
    }
```

Should we be doing more?